---
title: Citations
date: 2025-09-02 12:00:00 +0200
---

## Core software

### [Nextflow](https://pubmed.ncbi.nlm.nih.gov/28398311/)
> Di Tommaso P, Chatzou M, Floden EW, Barja PP, Palumbo E, Notredame C. Nextflow enables reproducible computational workflows. Nat Biotechnol. 2017 Apr 11;35(4):316-319. doi: 10.1038/nbt.3820. PubMed PMID: 28398311.

### [nf-core](https://pubmed.ncbi.nlm.nih.gov/32055031/)
> Ewels PA, Peltzer A, Fillinger S, Patel H, Alneberg J, Wilm A, Garcia MU, Di Tommaso P, Nahnsen S. The nf-core framework for community-curated bioinformatics pipelines. Nat Biotechnol. 2020 Mar;38(3):276-278. doi: 10.1038/s41587-020-0439-x. PubMed PMID: 32055031.

### [R](https://www.r-project.org)
> R Core Team (2017). R: A language and environment for statistical computing. R Foundation for Statistical Computing, Vienna, Austria.

### [Python](https://www.python.org/)
> Python Software Foundation. Python Language Reference, version 3.8.

## Pipeline tools

### [fastp](https://pubmed.ncbi.nlm.nih.gov/30423086/)
> Chen, S., Zhou, Y., Chen, Y., & Gu, J. (2018). fastp: an ultra-fast all-in-one FASTQ preprocessor. Bioinformatics, 34(17), i884–i890. doi:10.1093/bioinformatics/bty560. PubMed PMID: 30423086; PubMed Central PMCID: PMC6129281.

### [FastQC](https://www.bioinformatics.babraham.ac.uk/projects/fastqc/)
> Andrews, S. (2010). FastQC: A Quality Control Tool for High Throughput Sequence Data [Online].

### [MultiQC](https://pubmed.ncbi.nlm.nih.gov/27312411/)
> Ewels, P., Magnusson, M., Lundin, S., & Käller, M. (2016). MultiQC: summarize analysis results for multiple tools and samples in a single report. Bioinformatics (Oxford, England), 32(19), 3047–3048. https://doi.org/10.1093/bioinformatics/btw354

### [Bowtie](https://genomebiology.biomedcentral.com/articles/10.1186/gb-2009-10-3-r25)
> Langmead, B., Trapnell, C., Pop, M., et al. (2009) Ultrafast and memory-efficient alignment of short DNA sequences to the human genome. Genome Biol, 10, R25. https://doi.org/10.1186/gb-2009-10-3-r25

### [SAMtools](https://pubmed.ncbi.nlm.nih.gov/19505943/)
> Li H, Handsaker B, Wysoker A, Fennell T, Ruan J, Homer N, Marth G, Abecasis G, Durbin R; 1000 Genome Project Data Processing Subgroup (2009) The Sequence Alignment/Map format and SAMtools. Bioinformatics, 25(16), 2078-2079. doi: 10.1093/bioinformatics/btp352. PubMed PMID: 19505943; PubMed Central PMCID: PMC2723002

### [SeqKit](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0163962)
> Shen W, Le S, Li Y, Hu F (2016) SeqKit: A Cross-Platform and Ultrafast Toolkit for FASTA/Q File Manipulation. PLOS ONE 11(10): e0163962. https://doi.org/10.1371/journal.pone.0163962

### [BLAST](https://pubmed.ncbi.nlm.nih.gov/2231712/)
> Altschul, S.F., Gish, W., Miller, W., Myers, E.W., Lipman, D.J. (1990) Basic local alignment search tool. J. Mol. Biol. 215:403-410. PubMed
> Altschul SF, Gish W, Miller W, Myers EW, Lipman DJ (1990) Basic local alignment search tool. Journal of molecular biology, 215(3), 403–410. https://doi.org/10.1016/S0022-2836(05)80360-2

## R Packages

### [DESeq2](https://pubmed.ncbi.nlm.nih.gov/25516281/)
> Love MI, Huber W, Anders S. (2014). Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. Genome Biol, 15(12), 550. PubMed PMID: 25516281; PubMed Central PMCID: PMC4302049.

### [tidyverse](https://joss.theoj.org/papers/10.21105/joss.01686)
> Wickham H, Averick M, Bryan J, Chang W, McGowan L, François R, et al. (2019). Welcome to the tidyverse. J Open Source Softw, 4(43), 1686.

### [dendextend](https://academic.oup.com/bioinformatics/article/31/22/3718/240978)
> Galili T (2015). dendextend: an R package for visualizing, adjusting and comparing trees of hierarchical clustering. Bioinformatics, 31(22), 3718–3720.

### [plotly](https://cran.r-project.org/web/packages/plotly/index.html)
> Sievert C (2020). Interactive Web-Based Data Visualization with R, plotly, and shiny. Chapman and Hall/CRC, Florida. R package version 4.10.4, <https://CRAN.R-project.org/package=plotly>.

### [SARTools](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0157022)
> Varet, H., Brillet-Guéguen, L., Coppée, J. Y., & Dillies, M. A. (2016). SARTools: A DESeq2- and EdgeR-Based R Pipeline for Comprehensive Differential Analysis of RNA-Seq Data. PLoS One, 11(6), e0157022. https://doi.org/10.1371/journal.pone.0157022

### [DEGreport](https://www.bioconductor.org/packages/release/bioc/html/DEGreport.html)
> Pantano L (2025). DEGreport: Report of DEG analysis. doi:10.18129/B9.bioc.DEGreport, R package version 1.44.0, <https://bioconductor.org/packages/DEGreport>.

## Software packaging / containerisation

### [Anaconda](https://www.anaconda.com)

> Anaconda Software Distribution. Computer software. Vers. 2-2.4.0. Anaconda, Nov. 2016. Web.

### [Bioconda](https://pubmed.ncbi.nlm.nih.gov/29967506/)
> Grüning B, Dale R, Sjödin A, Chapman BA, Rowe J, Tomkins-Tinch CH, Valieris R, Köster J; Bioconda Team. Bioconda: sustainable and comprehensive software distribution for the life sciences. Nat Methods. 2018 Jul;15(7):475-476. doi: 10.1038/s41592-018-0046-7. PubMed PMID: 29967506.

### [Docker](https://dl.acm.org/doi/10.5555/2600239.2600241)
> Merkel D. Docker: lightweight linux containers for consistent development and deployment. Linux Journal, 2014(239), 2. doi: 10.5555/2600239.2600241.

### [Singularity](https://pubmed.ncbi.nlm.nih.gov/28494014/)
> Kurtzer GM, Sochat V, Bauer MW. Singularity: Scientific containers for mobility of compute. PLoS One. 2017 May 11;12(5):e0177459. doi: 10.1371/journal.pone.0177459; PubMed PMID: 28494014; PubMed Central PMCID: PMC5426675.
